migrations/2_apm.js

```diff
@@ -36,7 +36,7 @@ module.exports = function (deployer, network, accounts, arts = null) {
     console.log('Deployed APM at:', apmAddr)
 
     const apm = getContract('APMRegistry').at(apmAddr)
-    console.log('Kernel:', await apm.kernel())
+    console.log('Kernel:', await apm.kernel.call())
```

Comment:
Yes, they are for `testrpc` but I had some problems without using `geth` with our dao-kits, so now I'm putting them whenever I remember, so in case we finally move our tests to geth they will work.

Reply:
Hmm, weird, this should be handled by truffle (truffle-contract changes its interface based on whether the function is marked `constant` or not), rather than a testrpc / geth issue.